.circleci/config.yml

```diff
@@ -2,14 +2,28 @@ version: 2.1
 orbs:
   elixir: membraneframework/elixir@1
 
+executors:
+ machine_executor_amd64:
+   machine:
+     image: ubuntu-2204:2022.04.2
+   environment:
+     architecture: "amd64"
+     platform: "linux/amd64"
+
+jobs:
+  test:
+    executor: machine_executor_amd64
+    steps:
+      - checkout
+      - run: docker-compose -f docker-compose-integration.yaml run test
```

Comment:
I'm open to discussing this, but isn't that an overkill to use another container to run the tests? It complicates things a lot without much benefit

I thought you'd be using the same setup as in dev, i.e. start docker-compose with Jellyfish (this is what divo does, right?) and run `mix test` directly. This way you would get all the caching, and steps from the orb will be reusable.

This shows how to do this: https://circleci.com/docs/building-docker-images/#run-docker-commands-using-the-docker-executor

Reply:
In general, we have three options:
* run CI in docker - to run Jellyfish we have to start another docker so we end up having docker in docker, which turned out to be problematic in terms of network configuration
* run CI on bare machine - here we either have to install erlang and elixir to run tests or run tests in another container

The option with bare machine and having two containers was the easiest one to setup.

But as far as I understand this setup, we are still running docker in docker? We start `test` job from `integration`  docker compose, which starts tests in docker and those tests starts another docker compose `test` in the already running docker :thinking:

Reply:
We are not running docker inside docker. When running tests on the CI we use the `integration_test` environment,
which prevents Divo from launching docker:

```test_helper.exs
if Mix.env() != :integration_test do
  Divo.Suite.start()
end

ExUnit.start(capture_log: true)
```

Reply:
IMO, it's a little bit unintuitive that we start only `test` service because it is linked to jellyfish so Jellyfish will also be started.

I would say that we should just do `docker-compose up -f <file>` unless this is a common patter to do it the way it's currently done.

 It also looks like the `links` option is deprecated https://docs.docker.com/compose/compose-file/compose-file-v3/#links?